sklearn/base.py

```diff
@@ -320,7 +320,9 @@ def score(self, X, y, sample_weight=None):
         The coefficient R^2 is defined as (1 - u/v), where u is the regression
         sum of squares ((y_true - y_pred) ** 2).sum() and v is the residual
         sum of squares ((y_true - y_true.mean()) ** 2).sum().
-        Best possible score is 1.0, lower values are worse.
+        Best possible score is 1.0, 0.0 corresponds to the expected value
+        of the true y (i.e., chance), score can be arbitrarily negative
```

Comment:
This is not correct. I would rather say:



A constant model that always predicts the expected value of y irrespectively of the input features would get a R^2 score of 0.0.

(This is not "chance" level as it's a constant prediction based on the value of y but its a non-trivial baseline).